### PR TITLE
feat: align with kubernetes-client 4.2.0 and code generate boilerplate.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

**Comment by geoand** [comment]
@iocanel maybe we should merge #5 first so to ensure that CI is working?

**Comment by geoand** [comment]
Damn CircleCI... looks like it will need a rebase :(

**Comment by iocanel** [comment]
@geoand maybe but ci seems to run now.

**Comment by geoand** [comment]
Great! I guess it just took a while :)

**Comment by iocanel** [comment]
Not, sure if it was a matter of time. I had to change the branch policy to require circle ci.

File: client/src/main/java/me/snowdrop/servicecatalog/api/client/CodgeGen.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                @VelocityTransformation("/resource-operation.vm"),
+                @VelocityTransformation("/resource-handler.vm"),
+                @VelocityTransformation(value = "/resource-handler-services.vm", gather = true, outputPath = "META-INF/services/io.fabric8.kubernetes.client.ResourceHandler")
+        },
+        resources = {
+                @ResourceSelector("servicecatalog.properties")
+        }
+
+)
+public class CodgeGen {
```
Comment (geoand) [inline client/src/main/java/me/snowdrop/servicecatalog/api/client/CodgeGen.java]: CodeGen I guess ? :)

**Comment by geoand** [comment]
I don't really know what's going on with CI, but it seems to not be working properly - I don't see the run in the CircleCI UI at all...

**Comment by iocanel** [comment]
It must be the webhook or something

**Pull request merged**